Docs on RTOS (#668)
rtic-rs/cortex-m-rtic: book/en/src/preface.md

* v0.5.x go [here](/0.5).
* v0.4.x go [here](/0.4).

## Is RTIC an RTOS?

> A common question is wether RTIC is an RTOS or not, and depening on your background the

Reviewer: ```suggestion
A common question is whether RTIC is an RTOS or not, and depending on your background the
```

> answer may vary. From RTIC's developers the view is that RTIC is an hardware accelerated

Reviewer: ```suggestion
answer may vary. From RTIC's developers point of view; RTIC is a hardware accelerated
```

RTOS that utilizes the NVIC in Cortex-M MCUs to perform scheduling, rather than the more
classical software kernel.

> Another common view from the community is that RTIC is an concurrency toolkit as there

Reviewer: ```suggestion
Another common view from the community is that RTIC is a concurrency framework as there
```

Reviewer: To match our tagline: "A concurrency framework for building real-time systems" rtic.rs
